scala3doc/src/dotty/dokka/translators/ScalaSignatureProvider.scala

```diff
@@ -46,182 +46,163 @@ class ScalaSignatureProvider(contentConverter: CommentsToContentConverter)(using
 
   private def stylesIfDeprecated(m: Member): Set[Style] =
     if m.deprecated.isDefined then styles ++ Set(TextStyle.Strikethrough) else styles
-    
+
 
 object ScalaSignatureProvider:
   def rawSignature(documentable: Documentable, builder: SignatureBuilder): SignatureBuilder =
```

Comment:
Member?

Reply:
I will get rid of all `Documentable` after this PR is merged